src/code/z_effect_soft_sprite_old_init.c

```diff
@@ -168,9 +169,18 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_effect_soft_sprite_old_init/func_80029DBC.s")
 
-// EffectSsSolderSrchBall Spawn Functions
+EffectSsSolderSrchBall_Spawn(GlobalContext* globalCtx, Vec3f* pos, Vec3f* velocity, Vec3f* accel, s16 arg4,
+                             s16* linkDetected) {
```

Comment:
Is this function not exposed in `functions.h`?

Reply:
I guess I never added it cause it worked without a prototype, and thats the only place it gets called in the game.
I can add it tho for completeness sake

Reply:
Yeah it belongs in `functions.h` even if it's only used once

Reply:
added